src/Raven.Server/Web/System/ResourceNameValidator.cs

```diff
@@ -34,7 +35,7 @@ public class ResourceNameValidator
             "clock$"
         };
 
-        public const int WindowsMaxPath = 230;
+        public static readonly int WindowsMaxPath = (int)typeof(Path).GetField("MaxLongPath", BindingFlags.Static | BindingFlags.NonPublic).GetValue(null);
```

Comment:
No, better to just use this:

https://github.com/steveharter/dotnet_coreclr/blob/master/src/mscorlib/src/System/IO/PathInternal.cs#L34